blueocean-dashboard/src/main/js/util/logDisplayHelper.js

```diff
@@ -31,6 +31,9 @@ export const getNodesInformation = (nodes) => {
         const isFailingNode = errorNodes.indexOf(item.id) > -1;
         const isRunningNode = runningNodes.indexOf(item.id) > -1;
 
+        const hasLogs = item.actions ? item.actions
+                .filter(action => action._class === 'org.jenkinsci.plugins.workflow.support.actions.LogActionImpl').length > 0
```

Comment:
If the failure mode was to show up all log expanders, then it would be less of an issue...